src/glyphs.html

```diff
@@ -39,7 +39,7 @@
     }
   </style>
 
-  <script src="../build/vexflow-debug.js"></script>
+  <script src="../releases/vexflow-debug.js"></script>
```

Comment:
Hi, This `build/` is actually the correct path. It requires you to build the library before you can use it. This is the same for the tests and other `.html` files. Thanks.